### ISSUE TITLE
Testing checklist for `2022.11.1`

### ISSUE THREAD

**Issue by iameskild** [nebari-dev/nebari#1576]
# Testing Checklist

*Use/modify this checklist to capture the platform's core services that need to be manually tested .*

## Manual testing: core services

If the integration tests for all of the cloud providers are successful, that is a good sign!
However, the following core services still need to be manually validated (until we can automate them).

At minimum, the following services will need to be tested:
- [x] [Log into keycloak as root user](https://nebari-docs.netlify.app/how-tos/configuring-keycloak#change-keycloak-root-password)
  - [x] [Add a user](https://nebari-docs.netlify.app/how-tos/configuring-keycloak#adding-a-nebari-user)
- [x] [Log into conda-store and create](https://nebari-docs.netlify.app/tutorials/creating-new-environments)
  - [x] a conda environment in a shared namespace and,
  - [x] a conda environment in your personal namespace
- [x] [Launch dask-gateway cluster, test auto-scaler and](https://nebari-docs.netlify.app/tutorials/using_dask)
  - [x] [Validate that the dask-labextention is working](https://nebari-docs.netlify.app/tutorials/using_dask#step-5---viewing-the-dashboard-inside-of-jupyterlab)
- [x] [Create a basic CDS Dashboard](https://nebari-docs.netlify.app/tutorials/creating-cds-dashboard)
- [x] [Open VS-Code extension](https://nebari-docs.netlify.app/tutorials/using-vscode)
  - [x] [Add the Python extension](https://nebari-docs.netlify.app/tutorials/using-vscode#adding-extensions)
  - [x] [Create a `.py` file and run it](https://nebari-docs.netlify.app/tutorials/using-vscode#running-python-code)

**Comment by iameskild** [comment]
During testing, I ran into a two dask issues. 
- A recent PR to fix one dask bug broke how the dask worker node group selection
  - Fixed in [this PR](https://github.com/nebari-dev/nebari/pull/1577)
- The dask dashboard doesn't appear to work with Bokeh >= 3.0.0
  - Fixed by bumping the dask/dask-gateway versions [here](https://github.com/conda-forge/nebari-dask-feedstock/pull/1)

We also recently merged a PR to include the conda-store jupyterlab extension but we decided today that we would hold off including this in the `2022.11.1` release. As such, I have removed that package from the JupyterLab docker image in [this PR](https://github.com/nebari-dev/nebari-docker-images/pull/42).
